js/ui/drop_down_box.js

```diff
@@ -290,7 +290,7 @@ const DropDownBox = DropDownEditor.inherit({
             position: extend(this.option('popupPosition'), {
                 of: this.$element(),
             }),
-            onKeyboardHandled: opts => this.option('focusStateEnabled') && this._popupElementTabHandler(opts),
+            onKeyboardHandled: opts => focusStateEnabled && this._popupElementTabHandler(opts),
```

Comment:
Will this function work correctly after changing the `focusStateEnabled` option at runtime?

Reply:
fixed